### ISSUE TITLE
[Bug] search box disappear

### ISSUE THREAD

**Issue by SynapseScribe** [henrypp/simplewall#1790]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

3.8.2

### Windows version

win 11

### Steps to reproduce

open main window
search box missing sometimes
resizing the window makes the search box reappear

issue was introduced in the past 6 months or so

### Expected behavior

search box should be in the right top corner

### Actual behavior

search box missing

**Comment by Spheres-cu** [comment]
Still happen in version 3.8.7 Windows 10 22H2

**Comment by henrypp** [comment]
now, problem is gone

https://github.com/henrypp/simplewall/issues/2032#issuecomment-3213630314

**Comment by henrypp** [comment]
@JIGASHI u stupid or it fun?

https://github.com/henrypp/simplewall/issues/1790#issuecomment-3340081453

**Comment by Spheres-cu** [comment]
> [@JIGASHI](https://github.com/JIGASHI) u stupid or it fun?
> 
> [#1790 (comment)](https://github.com/henrypp/simplewall/issues/1790#issuecomment-3340081453)

Да, на самом деле в этой версии больше нет ошибки с поисковой строкой, но при генерации бинарного файла забыли поставить цифровую подпись. Windows Defender выдаёт предупреждение по этому поводу.

Я думаю, что, хотя это и незначительная деталь в коде, она заслуживает версии со всем необходимым: установщиком, портативной версией и цифровой подписью. Проблема несерьёзная, и, судя по всему, её было легко исправить, но с ней сталкиваются несколько пользователей, и дело не в том, что они глупые или особенно забавные. Обычные пользователи, или, возможно, те, у кого мало времени, чтобы разбираться в приложении, просто обновляют его и не скачивают бинарный файл из не очень заметного поста на GitHub.

Извините за длинный пост, но это моё мнение.

С уважением и за отличную работу.

Примечание: я не говорю по-русски; я говорю по-испански и немного по-английски. 😅

**Comment by Spheres-cu** [comment]
Que bueno que hable español ! 😀

Es que Defender viene por defecto en todos los OS Windows y lo prefiero antes que un antivirus de tercero, pero yendo al caso de simplewall (que alguna forma interactúa con Defender ya que tiene la opción de eliminar las reglas de este para suplantar las suyas 😂 big balls! )

No es obligatorio ni tan urgente el bug del cuadro de búsqueda pero no se va a caer ninguna compañía multimillonaria y menos simplewall por una versión con ese bug corregido (I'm not joking! 😡... Mentira es broma 🙃) 

Usted es un excelente desarrollador al igual que su aplicación pero esa funcionalidad de la búsqueda aunque no lo crea la usan muchos usuarios yo incluido... So a please please for a corrected version, por fa?

Mis saludos afectuosos!

**Comment by ennu1** [comment]
The search bar still disappears. I keep having to toggle it on and off. :/

**Comment by Spheres-cu** [comment]
https://github.com/henrypp/simplewall/releases/tag/v.3.9 
Still the bug:

<img width="958" height="506" alt="Image" src="https://github.com/user-attachments/assets/1533112e-0a03-4163-a031-9b58000ec6e1" />

**Comment by henrypp** [comment]
@Spheres-cu ye, not completely fixed

**Comment by Spheres-cu** [comment]
> [@Spheres-cu](https://github.com/Spheres-cu) ye, not completely fixed

Fixed in https://github.com/henrypp/simplewall/commit/e9e45f44182de9cf65438ab7756524c78470dad9

Thanks! @henrypp

**Comment by SynapseScribe** [comment]
seems to be fixed on 3.9.1. thank you
